tsconfig.dev.json

```diff
@@ -2,5 +2,9 @@
 {
   "extends": "./tsconfig.json",
 
-  "include": ["**/scripts/**/*.ts"],
+  "include": ["**/scripts/**/*.ts", "jest/**/*.ts"],
+
+  "compilerOptions": {
+    "types": ["node", "jest"],
```

Comment:
Are you sure you want to add `node` typings everywhere? Wont it sometimes clash between browser/node, just like we had with `Request`, `Event` and similar?

Reply:
Ah, gotcha. Perfectly fine then.